formats/definitions.json

```diff
@@ -526,6 +526,30 @@
           "$ref": "#/definitions/absolute_path"
         }
       }
+    },
+    "manual_organisations": {
+      "description": "A manual’s organisations. TODO: Switch to use organisations in links",
```

Comment:
I think we might be able to drop these already.  hmrc manuals don't look like they have the organisation in the details hash at all (but do have it in the links) and govuk manuals appear to have both.  We control govuk manuals so we could drop it and republish things relatively easily.  The frontend has already been updated to prefer the links.organisation over the details.organisation (see: https://github.com/alphagov/manuals-frontend/pull/169).